docs/src/main/asciidoc/amazon-lambda.adoc

```diff
@@ -259,75 +226,18 @@ NB: With gradle, to build the uber-jar execute: ./gradlew quarkusBuild --uber-ja
 
 The https://docs.aws.amazon.com/serverless-application-model/latest/developerguide/serverless-sam-cli-install.html[AWS SAM CLI] allows you to run your lambdas locally on your laptop in a simulated Lambda environment.  This requires https://www.docker.com/products/docker-desktop[docker] to be installed.
 
-To execute your lambda function with the
-`java8` runtime, create a `sam.jvm.yaml` template as below:
-----
-AWSTemplateFormatVersion: '2010-09-09'
-Transform: AWS::Serverless-2016-10-31
-
-Globals:
-  Function:
-    Timeout: 5
-Resources:
-  QuarkusSam:
-    Type: AWS::Serverless::Function
-    Properties:
-      Handler: io.quarkus.amazon.lambda.runtime.QuarkusStreamHandler::handleRequest
-      Runtime: java8
-      CodeUri: {path to *-runner.jar}
-      MemorySize: 128
-      Policies: AWSLambdaBasicExecutionRole
-----
-
-Run the following SAM CLI command to test your lambda function with the `Java8` runtime and `sam.jvm.yaml` template:
-----
-sam local invoke --template sam.jvm.yaml --event {json test file}
-----
-
-The native executable can be tested via the `provided` runtime defined in a `sam.native.yaml` template:
-----
-AWSTemplateFormatVersion: '2010-09-09'
-Transform: AWS::Serverless-2016-10-31
-
-Globals:
-  Function:
-    Timeout: 5
-Resources:
-  QuarkusSam:
-    Type: AWS::Serverless::Function
-    Properties:
-      Handler: not.used.in.provided.runtimei
-      Runtime: provided
-      CodeUri: {path to function.zip}
-      MemorySize: 128
-      Policies: AWSLambdaBasicExecutionRole
-      Timeout: 15
-      Environment:
-        Variables:
-          DISABLE_SIGNAL_HANDLERS: true
-----
-
-
-When using the `provided` runtime, AWS requires a bootstrap script to trigger the native executable. Bundle the `bootstrap` script along with the *-runner executable.
+A starter template has been generated for both JVM and native execution modes.
 
-i.e., create a zip file called `function.zip` with:
+Run the following SAM CLI command to locally test your lambda function:
 
-* bootstrap
-* {projectname-version}-runner
-
-Example `bootstrap` script to load the native executable:
+[source]
 ----
-#!/usr/bin/env bash
-
-RUNNER=$( find . -maxdepth 1 -name '*-runner' )
-if [[ ! -z "$RUNNER" ]]
-then
-    $RUNNER
-fi
+sam local invoke --template sam.jvm.yaml --event {json test file}
 ----
 
-To invoke the native `provided` runtime with the `sam.native.yaml` template, use the following command:
+The native executable can also be locally tested the `sam.native.yaml` template:
```

Comment:
This sentence doesn't seem right

Reply:
updated s/executable/image/.  better?

Reply:
Now I see what I think is missing: `tested the sam.native.yaml` I assume should be `tested using the sam.native.yaml`

Reply:
FTR, `native executable` is the wording we agreed on.